app/code/Magento/MediaGalleryUi/view/adminhtml/web/js/image/image-edit.js

```diff
@@ -223,6 +223,16 @@ define([
             }
 
             return true;
+        },
+
+        /**
+         * Remove cached image details in edit form
+         *
+         * @param {String} id
+         * @param {String} tags
+         */
+        removeCached: function (id, tags) {
```

Comment:
I would simply remove cached image and rely on reloading from the backend to make sure frontend and backend are on the same page